src/commands.c

```diff
@@ -3338,7 +3360,8 @@ const char ffa_um_init[] =
 	"k_lockmax 0\n"				// no effect in ffa
 	"k_overtime 1\n"			// time based
 	"k_exttime 5\n"				// overtime 5mins
-	"k_mode 3\n";				//
+	"k_mode 3\n"				//
+	"k_bzk 1\n";				// berzerk mode on ;)
```

Comment:
really? :/

Reply:
For FFA? Why not?
And this is (as far as I understood) just the 'default' config. Each server admin could create his own.

But if you are really against it, just set it to 0. No probs.